scripts/make_attribution.sh

```diff
@@ -44,12 +44,13 @@ source "$REPO_ROOT/scripts/attribution_helpers.sh"
 function build::attribution::generate(){
     cd $REPO_ROOT
     build::common::use_go_version "$GOLANG_VERSION"
-    go mod vendor
+    local -r GO=$(build::common::get_go_path "$GOLANG_VERSION")/go
```

Comment:
Was this necessary? In theory, `use_go_version` should take care of the PATH env var, so you can simply call `go`. That said, this doesn't hurt I guess.